daemon/srm.cpp

```diff
@@ -271,15 +271,16 @@ int32_t SrmTable::VerifySignature(
     r = BN_bin2bn(pR, DSA_SIG_LENGTH, nullptr);
     s = BN_bin2bn(pS, DSA_SIG_LENGTH, nullptr);
 
-    if(DSA_SUCCESS != DSA_SIG_set0(sig, r, s)) 
+#if OPENSSL_VERSION_NUMBER < 0x10100000L
```

Comment:
Can you just explain a little about why this change is here?

Reply:
This function is part of encapsulation under "OPENSSL_VERSION_NUMBER < 0x10100000L) per specification (see line 81), without which the code will not compile.